src/index.js

```diff
@@ -17,12 +20,13 @@ app.use(cors());
 
 app.use(require('morgan')('dev'));
 
-app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
Hello, is there any reason why you chose to change extended to `false`? I think leaving it as `true` would allow the use of rich text besides only arrays and strings. Please check the **urlencoded** part [here](https://www.npmjs.com/package/body-parser) for more information

Reply:
No reason, just trying out things. Forgot to revert afterwards